modules/services/pizauth.nix

```
@@ -0,0 +1,195 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  inherit (lib) mkOption types;
+
+  cfg = config.services.pizauth;
+in
+{
+  meta.maintainers = [ lib.hm.maintainers.swarsel ];
+
+  options.services.pizauth = {
+    enable = lib.mkEnableOption ''
+      Pizauth, a commandline OAuth2 authentication daemon.
```

```suggestion
      Pizauth, a commandline OAuth2 authentication daemon
```
`mkEnableOption` will already include an ending punctuation.